packages/typescript-fetch-client/templates/api.hbs

```diff
@@ -27,7 +27,7 @@ export const {{className name}}ApiFetchParamCreator = function (configuration?:
 	{{>frag/validateParameter operation=..}}
 	{{/with}}
 			let localVarPath = `{{{../path}}}{{{path}}}`{{#each pathParams}}
-				.replace('{{safe '{'}}{{{serializedName}}}{{safe '}'}}', encodeURIComponent(String({{identifier name}}))){{/each}};
+				.replace('{{safe '{'}}{{{serializedName}}}{{safe '}'}}', String({{identifier name}})){{/each}};
```

Comment:
This doesn't feel equivalent. If there's a `/` in the component we will no longer encode it. I'm not sure what the contract with OpenAPI is here... but this feels a bit like we're moving the encoding to the wrong place.

Does `encodeURIComponent` do a bad job? We could replace incorrectly encoded things after using it if we need to.

Reply:
On MDN we see the following example, which seems like something we should support: https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname

> Note that some systems use the ; and = characters to delimit parameters and parameter values applicable to a path segment. For example, with the URL https://example.org/users;id=42/tasks;state=open?sort=modified, a system might extract and use the path segment parameters id=42 and state=open from the path segments users;id=42 and tasks;state=open.

I'm liking the above change. I'll amend the PR.